e2e/jail/jail_rpc_test.go

```diff
@@ -39,6 +39,8 @@ func (s *JailRPCTestSuite) TestJailRPCSend() {
 	s.StartTestBackend(params.RopstenNetworkID)
 	defer s.StopTestBackend()
 
+	s.EnsureNodeSync()
```

Comment:
Should it be moved to `StartTestBackend()`? We probably want start tests after ensuring syncronization anyways?

Reply:
You mean calling it always inside of `StartTestBackend()`? We have calls of the start where a node sync blocks. Not yet sure if it's the individual node manager or `LightEthereumService`. Here I had to remove the syncing after tests.

Reply:
@divan not all `StartTestBackend` calls are followed by blockchain syncing and that's why we haven't done it so far. Anyway, leaving this investigation to @themue.